core/tauri/src/ipc/authority.rs

```diff
@@ -84,6 +85,93 @@ impl RuntimeAuthority {
     }
   }
 
+  #[doc(hidden)]
```

Comment:
does this need to be hidden also? and have this internal name? why not just expose it?

Reply:
not stable yet.. maybe we move it under the unstable feature flag, but I don't like the idea of allowing devs to allow an specific command, they should go through the whole capabilities > permissions folow, this was just added for our examples which can't use a build script so they do not go through our ACL definitions
cc @tweidinger